plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -191,9 +205,14 @@ class ContainerServiceFormSection extends Component {
 
           <FormGroup
             className="column-3"
-            required={true}
```

Comment:
I moved the `required` prop to the `FieldLabel` which is now responsible for displaying the asterisk. I missed the fact that it passed the `required prop to the input elements, so I'll have to revisit that...